dist/21-526EZ-schema.json

```diff
@@ -1441,6 +1441,10 @@
             "Philippine Islands",
             "U.S. Minor Outlying Islands"
           ]
+        },
+        "zipCode": {
+          "type": "string",
+          "pattern": "^\\d{5}(?:[-\\s]\\d{4})?$"
```

Comment:
What do you think about making the separator `[-\s]` optional (`[-\\s]?`)? It's weird, but it would give the user the most freedom and we don't _really_ care at the end of the day, do we? We'll just be stripping out any separator and splitting it up by the first five and last four.

If you don't want to make that change here, that's totally fine by me. It was just a passing thought.